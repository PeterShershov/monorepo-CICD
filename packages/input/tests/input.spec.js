import React from "react";
import renderer from "react-test-renderer";

import Input from "../input";

describe("Input component", () => {
  test("Input changes style when active", () => {
    const cmp = renderer.create(<Input value="TEST" />);
    expect(cmp).toMatchSnapshot();
  });
});
