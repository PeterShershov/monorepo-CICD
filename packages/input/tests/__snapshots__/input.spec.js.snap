// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component Input changes style when active 1`] = `
<input
  className="input input_active"
  onChange={[Function]}
  value="TEST"
/>
`;
